scripts/scr_vehicle_count/scr_vehicle_count.gml

```diff
@@ -1,43 +1,47 @@
 // Counts the number of total vehicles for the player
+// TODO this should be function defined in Chapter struct, or something similar
 function scr_vehicle_count(role, star_system_num) {
 
-	var count=0,com=0,coom=-999;
+    var vehicle_count = 0,
+        company_number = 0,
+        company_number_derived_from_star_system_num = -999; // TODO refactor star_system_num to actually represent something logical
 
-	if (star_system_num="0") then coom=0;
-	if (star_system_num="1") then coom=1;
-	if (star_system_num="2") then coom=2;
-	if (star_system_num="3") then coom=3;
-	if (star_system_num="4") then coom=4;
-	if (star_system_num="5") then coom=5;
-	if (star_system_num="6") then coom=6;
-	if (star_system_num="7") then coom=7;
-	if (star_system_num="8") then coom=8;
-	if (star_system_num="9") then coom=9;
-	if (star_system_num="10") then coom=10;
+    if (star_system_num = "0") then company_number_derived_from_star_system_num = 0;
```

Comment:
I don't like this if chain, but it has to stay for now

Reply:
We can improve it later on on another refactor sprint, smaller than this one in the end